components/dashboard/src/projects/Projects.tsx

```diff
@@ -122,27 +122,46 @@ export default function () {
                                         }]} />
                                     </div>
                                 </div>
-                                <p>{toRemoteURL(p.cloneUrl)}</p>
+                                <Link to={`/`}>
+                                    <p className="hover:text-gray-600 dark:hover:text-gray-400 dark:text-gray-500">{toRemoteURL(p.cloneUrl)}</p>
+                                </Link>
+                            </div>
+                            <div className="h-10 px-6 py-1 text-gray-400 text-sm">
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Branches
+                                    </Link>
+                                </span>
+                                <span className="mx-2 my-auto">·</span>
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Prebuilds
+                                    </Link>
+                                </span>
                             </div>
-                            <div className="h-1/4 px-6 py-1"><p>__ Active Branches</p></div>
                         </div>
-                        <div className="h-1/6 px-6 border rounded-b-xl dark:border-gray-800 bg-gray-200 cursor-pointer" onClick={() => viewAllPrebuilds(p)}>
+                        <div className="h-10 px-4 border rounded-b-xl dark:border-gray-800 bg-gray-100 border-gray-100 dark:bg-gray-800" onClick={() => viewAllPrebuilds(p)}>
                             {lastPrebuilds.get(p.id)
-                                ? (<div className="flex flex-row space-x-3 h-full text-sm">
-                                    <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
-                                        &nbsp;
-                                    </div>
-                                    <div className="my-auto">{lastPrebuilds.get(p.id)!.branch}</div>
-                                    <div className="my-auto text-gray-400">{moment(lastPrebuilds.get(p.id)!.startedAt, "YYYYMMDD").fromNow()}</div>
-                                    <div className="my-auto text-gray-400 flex-grow text-right">View All ⟶</div>
+                                ? (<div className="flex flex-row h-full text-sm justify-between">
+                                    <Link to={`/`} className="flex my-auto group space-x-2">
+                                        <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
```

Comment:
**issue**: Ideally this would also use the prebuild status icons used in the _Branches_ and _Prebuilds_ pages and reflect the current state of the lates prebuild (`PENDING`, `RUNNING`, `FAILED`, etc).

| BEFORE (`READY`)| AFTER (`READY`) | AFTER (`RUNNING`) |
|-|-|-|
| <img width="432" alt="Frame 315" src="https://user-images.githubusercontent.com/120486/128593877-4549e58d-d2bb-4b12-822e-b33dc4e324ea.png"> | <img width="432" alt="Frame 316" src="https://user-images.githubusercontent.com/120486/128593881-895ce2e7-91ce-4380-91f8-c8ef1f123c8d.png"> | <img width="432" alt="Frame 317" src="https://user-images.githubusercontent.com/120486/128593963-e67f61e5-794d-411f-acc3-f511d20e70ce.png"> |

Reply:
FYI, you can import these icons at the top of the file with:

```
import StatusDone from "../icons/StatusDone.svg";
import StatusPaused from "../icons/StatusPaused.svg";
import StatusRunning from "../icons/StatusRunning.svg";
```

And then use them as images with something like this:

```suggestion
                                        <img className="h-3 w-3" src={StatusDone} />
```

Reply:
SUCCESS! Thanks @jankeromnes! ⭐